compiler/AST/stmt.cpp

```diff
@@ -62,6 +64,35 @@ bool Stmt::isStmt() const {
 *                                                                             *
 ************************************** | *************************************/
 
+VisibilityStmt::VisibilityStmt(AstTag astTag): Stmt(astTag) {
+}
+
+VisibilityStmt::~VisibilityStmt() {
+}
+
+bool VisibilityStmt::isVisibilityStmt() const {
```

Comment:
I'm having trouble understanding the point of this method. It doesn't seem to be used, either. Can you remove it?